actors/market/tests/market_actor_test.rs

```diff
@@ -96,6 +96,22 @@ fn simple_construction() {
     assert_eq!(state_data.last_cron, EPOCH_UNDEFINED);
 }
 
+#[test]
+fn label_cbor() {
```

Comment:
can you add some tests with bytes that are not valid utf8 strings, and ones with nul bytes in them?

Reply:
@dignifiedquire I'm taking this PR over.  lmk if you think I'm missing any cases.  I think I can only test non utf8 on decode because rust String s can't take on non-utf8 values in the first place.

Reply:
you can make an evil string by using String::from_utf8_unchecked which allows you to test a degenerate case